src/condor_utils/dagman_utils.cpp

```diff
@@ -944,21 +975,20 @@ DagmanUtils::create_lock_file(const char *lockFileName, bool abortDuplicates) {
 	if (result == 0 && abortDuplicates) {
 		int status;
 		if (ProcAPI::confirmProcessId(*procId, status) != PROCAPI_SUCCESS) {
-			dprintf(D_ERROR, "Warning: ProcAPI::confirmProcessId() failed; %d\n",
-			        status);
+			print_error("Warning: Failed to confirm process ID (%d)\n", status);
 		} else if ( ! procId->isConfirmed()) {
-			dprintf(D_ERROR, "Warning: ProcessId not confirmed unique\n");
+			print_error("Warning: ProcessId not confirmed unique\n");
```

Comment:
Not sure it make a lot of sense to have an error prefixed by "Warning".  You could maybe rephrase it as something like "Ignoring error that ProcessID not confirmed unique", I guess?

Reply:
Added the `ignoring error that` portion to message and made message print to non-error stream